scroll/scroll_readme.md

```diff
@@ -0,0 +1,158 @@
+| 400         | "Found JSON elements which are not allowed [...]" | See Limits above for list of allowed elements in Query |
+| 400         | "[...] query_string.allow_leading_wildcard not allowed to be true" | |
+| 400         | "[...] 'wildcard' queries are not allowed to start with '*' or '?'" | |
+| 400         | "ScrollId and query parameter can not co-exist in request" | |
+| 400         | "Scroll can't exceed [...]" | |
+| 400         | "[...] contains a wildcard search with leading wildcard which is not allowed [...]" | |
+| 400         | "Query has failed. Elasticsearch Response: [...]" | Elasticsearch failed running this search request. See the Elasticsearch response to understand what went wrong. Most likely the syntax of the search request does not conform with the search API DSL - see link above for full documentation |
+| 400         | "Query syntax (in 'query' field) is invalid: [...]" | |
+| 400         | "This search can't be executed: [...]. Please contact customer support for more details" | Something unexpected in the query prevented him from running. Contact our customer support for help | 
+| 403         |  | Your account is not allowed to use Scroll API. Please contact support/sales to enable this feature for you|
+| 429         |  | Exceeded your account rate limit of searches |
+| 429         | "Your account is allowed to run up to __ concurrent queries via the API" | Your account is allowed to run up to X concurrent queries via the Scroll API. This error means you've exceeded this limit |
+| 503         |  | Our service is temporarily unavailable. You should try again when that happens. |
+| All other codes | A message that explain the specific error | Query failures |
+----------------------
+
+
+## Example
+
+### Initial Search
+Create a file named `query.json` and place the following query in it:
+```json
+{
+	"size": 1,
+	"scroll": "1m",
+	"query": {
+		"bool": {
+			"must": [{
+				"range": {
+					"@timestamp": {
+						"gte": "now-5m",
+						"lte": "now"
+					}
+				}
+			}]
+		}
+	}
+}
+```
+This query returns scroll_id and 1 document from the last 5 minutes
+
+Run the following command:
+```bash
+curl -XPOST 'https://api.logz.io/v1/scroll'  --header "X-API-TOKEN: your-api-access-token" --header "Content-Type: application/json" --data-binary @query.json
+```
+
+Make sure to replace `your-api-access-token` with the token you created as instructed in License section
+
+Example result:
+
+```json
+{
+  "code": 200,
+  "scrollId": "DnF1ZXJ5VGhlbkZldGNoAgAAAAAAN0JtFkNnX3lDWUFqU0R5TkNqQ3JPdm55ancAAAAAADdCbhZDZ195Q1lBalNEeU5DakNyT3ZueWp3",
+  "hits": {
+  	"total":20,
+    "max_score":1.0,
+	"hits":
+	  [{
+	  	"_index":"index_v9",
+	  	"_type":"t1",
+	  	"_score":1.0,
+	  	"_source":{
+	  		"message":"hello world",
+	  		"type":"t2",
+	  		"@timestamp":"2017-11-08T12:32:38.603Z"
+	  	}
+	  }]
+	}
+}
+```
+
+### Next Batch
+Create a file named `batch.json` and place the following in it:
```

Comment:
Need to note that the user should replace the scroll id here to the one which was returned in the response for the first request.